epub33/rs/index.html

```diff
@@ -1072,14 +1072,14 @@ <h4>The <code>rendition:page-spread-*</code> Properties</h4>
 							regular and center-spread pages.</p>
 					</div>
 
-					<p id="page-layout-both">When a <a href="#def-layout-reflowable">reflowable</a> spine item follows a
+					<p id="page-layout-both" data-tests="#page-layout-both">When a <a href="#def-layout-reflowable">reflowable</a> spine item follows a
```

Comment:
As I noted in the test CL, it doesn't make sense to me that this existing ID, or the test, is called page-layout-both. See comment there, and probably rename.

Reply:
This was the anchor in the spec originally, so I've opted not to change it, but we never really "named" this feature so I'm not sure what to call this anchor/test.

Reply:
Yeah, fine not to change the anchor, but we shouldn't repeat the bad example in the test. How about #fxl-page-spread-flow-undefined?

You also *could* insert a <span> or an <a> or something with the old anchor (so as not to break links) and then rename the <p> id to match.